src/BladeError/BladeError.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace HelsingborgStad\BladeError;
+
+use Throwable;
+use Exception;
+use HelsingborgStad\BladeService\BladeService as BladeErrorRenderer;
+
+/**
+ * Class BladeServiceInstance
+ *
+ * This class provides functionality for managing Blade views.
+ */
+class BladeError implements BladeErrorInterface
+{
+  public function __construct(private Throwable $e, private array $viewPaths = [], private array $viewData = []){}
+
+  public function print(): void 
+  {
+    $errorTemplateViewPath = __DIR__ . "/../../views/";
+    $renderer = new BladeErrorRenderer([$errorTemplateViewPath]);
```

Comment:
Consider passing BladeServiceInterface as a dependency to the constructor to make this testable.